src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.scss

```diff
@@ -9,6 +9,10 @@
   padding: 75px 20px 10px 20px;
   height: 100%; // may need to change this later
   width: 100%;
+  max-width: 100vw;
+  position: absolute;
```

Comment:
I don't like this idea. Rather, let's fix the height on the `.standardAppLayout` class:

```
.standardAppLayout {
  height: calc(100vh - 159px);
}
```
where 159px is header height plus appBar height; and then geneView will take 100% of the allotted height.

Reply:
That's done